go/libraries/doltcore/sqle/index/prolly_fields.go

```diff
@@ -91,12 +91,27 @@ func GetField(ctx context.Context, td val.TupleDesc, i int, tup val.Tuple, ns tr
 			err = json.Unmarshal(buf, &doc.Val)
 			v = doc
 		}
+	// TODO: eventually remove this, and only read GeomAddrEnc
 	case val.GeometryEnc:
```

Comment:
Are any tests going to hit this logic to see if it works?

Reply:
added some in prolly_field_tests.go